docs/sources/send-data/promtail/cloud/eks/_index.md

```diff
@@ -243,7 +244,7 @@ If you want to push this further you can check out [Joe's blog post][blog annota
 [kubelet]: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#:~:text=The%20kubelet%20works%20in%20terms,PodSpecs%20are%20running%20and%20healthy.
 [blog events]: https://grafana.com/blog/2019/08/21/how-grafana-labs-effectively-pairs-loki-and-kubernetes-events/
 [labels post]: https://grafana.com/blog/2020/04/21/how-labels-in-loki-can-make-log-queries-faster-and-easier/
```

Comment:
Unrelated to this PR, the https://grafana.com blog links should be just absolute path partial URLs (remove the scheme and hostname) so that they correctly resolve in preview builds and not just on the production website.